erpnext/crm/report/lead_owner_efficiency/lead_owner_efficiency.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2016, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+frappe.require("assets/erpnext/js/financial_statements.js", function() {
```

Comment:
i don't think this require is needed (if you are not using any special JS functions (?)).